Documentation/dev-tools/kfence.rst

```diff
@@ -40,11 +40,12 @@ guarded by KFENCE. The default is configurable via the Kconfig option
 ``CONFIG_KFENCE_SAMPLE_INTERVAL``. Setting ``kfence.sample_interval=0``
 disables KFENCE.
 
-With the Kconfig option ``CONFIG_KFENCE_NUM_OBJECTS`` (default 255), the number
-of available guarded objects can be controlled. Each object requires 2 pages,
-one for the object itself and the other one used as a guard page; object pages
-are interleaved with guard pages, and every object page is therefore surrounded
-by two guard pages.
+The KFENCE memory pool is of fixed size, and if the pool is exhausted, no
```

Comment:
It just came to my mind that KFENCE adds some implicit memory cost by splitting bigger pages and potentially adding another level to certain page table entries.
Not sure it is worth mentioning though.

Reply:
It's quite complicated to explain, without going into architecture-specific details. Since it's architecture-specific, we can probably ignore it in Documentation, and if it's important, can just mention it in the various <asm/kfence.h> files.